app/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "headlamp",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "copy-server-binary": "cd ../ && make backend && cp -r ./backend/server ./app/electron/",
+    "start-electron": "npm run copy-server-binary && cd ../ && make run-backend  & cd ../frontend/ && export BROWSER=none && npm start",
+    "build-electron": "npm run copy-server-binary && cd ../frontend/ && npm run build && cp -r ../app/electron/. build/electron",
+    "package-electron": "mkdir -p build && cp -r ../frontend/build/ ./build && electron-builder build --mac --win --linux -c.extraMetadata.main=build/electron/main.js --publish never",
+    "serve-electron-dev": "export ELECTRON_START_URL=http://localhost:3000 && cd ../frontend/ && cp -r ../app/electron/. electron && electron ."
```

Comment:
Nitpick: I don't think we need to use the `electron` keyword in the commands now that we have this separated, i.e. this whole package.json file refers to the electron app. So unless the command clashes with an existing one (I guess `start` and `build`), don't add electron. i.e. maybe simply `package`, `serve-dev` or just `dev`.